pkgs/games/steam/fhsenv.nix

```diff
@@ -223,9 +222,7 @@ in buildFHSEnv rec {
     zlib
     atk
     cairo
-    freetype
     gdk-pixbuf
-    fontconfig
```

Comment:
> drop game specific libraries now duplicated with steam own

To be clear, this means that steam has added these libraries to it's runtime?

Reply:
We already have them in the list above.